builders/toolkit/substrate-api/dev-env/index.md

```diff
@@ -1,8 +1,28 @@
 ---
 title: Substrate Development Environments
 description: Check out the Substrate development environments, which can be used to develop, test, and debug a Tanssi appchain in a local development environment.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
 ---
+
+# Set Up Your Workspace
+
+Establishing an effective development environment is crucial for building, testing, and deploying smart contracts efficiently. A development environment provides the necessary tools and frameworks to:
+
+- Write and compile smart contracts using standardized languages.
+- Simulate blockchain interactions locally for testing and debugging.
+- Manage dependencies and automate workflows for faster deployments.
+
```

Comment:
This looks like it was probably just copy pasted from the EVM section, but we need to modify it for Substrate. No smart contracts